### ISSUE TITLE
Request: Enabling support for x number of layers only.

### ISSUE THREAD

**Issue by Capital-M** [alexrj/Slic3r#3128]
Can an option be added to turn off support beyond a specified height or number of layers please?

I have a concave-bottomed holder for a captive nut. This needs to be supported due to the text. 
![lower](https://cloud.githubusercontent.com/assets/15818365/11121574/95754618-894e-11e5-95b8-28c98fd9e27d.png)

But the area where the nut is inserted needs to be left clear.
![upper](https://cloud.githubusercontent.com/assets/15818365/11121529/5130b816-894e-11e5-8262-760cfc5e08a7.png)

I understand that I may be the only person who wishes for this feature, and I could do away with the text, and therfore support would not be necessary.

**Comment by markwal** [comment]
You should be able to print that pretty well without support at all. The top of those letters are pretty short bridges.

The request might be interesting for other reasons though.

**Comment by Capital-M** [comment]
Without support the letters print in the air.
![no_support](https://cloud.githubusercontent.com/assets/15818365/11142744/9d261f24-89e7-11e5-8b71-d13aa0194121.png)
![section](https://cloud.githubusercontent.com/assets/15818365/11142817/f9901666-89e7-11e5-8ef7-f9d41d910619.png)

**Comment by markwal** [comment]
Ah, that is more clear.

**Comment by pyrophreek** [comment]
At the moment, you can solve this by using a modifier mesh. Create a cube that is larger than the object in x and y, but just as tall as the letters, and make sure it is positioned so that when both are imported using the same coordinate references, they overlap. Then go into slicer and load your object, click settings, and load the cube as a modifier mesh. Enable support for the modifier mesh, but disable it for the rest of the print.

**Comment by pyrophreek** [comment]
See this post for a slightly more detailed explanation: https://groups.google.com/forum/m/#!topic/3d604/WaY_HFeduPs

**Comment by lordofhyphens** [comment]
Actually, you can't disable supports over a modifer mesh currently. I think that this is something that can be done, just that it hasn't been implemented.

@alexrj @bubnikv  any thoughts?

**Comment by bubnikv** [comment]
Yes, modifier volumes over supports are on my plate and it has a high priority, I did not start yet though, pulling straw after straw.
